packages/react-composites/tests/server/server.ts

```diff
@@ -10,20 +10,21 @@ const app = express();
 export const createTestServer =
   (props: { appDir: string; serverUrl: string }) =>
   // eslint-disable-next-line no-empty-pattern, @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/explicit-function-return-type
-  async ({}, use: (r: string) => Promise<void>) => {
-    await startServer(props.appDir);
+  async ({}, use: (r: string) => Promise<void>, workerInfo) => {
+    const port = 3000 + workerInfo.workerIndex;
```

Comment:
This is a neat fix for this issue where each worker spawns their own server! Can you file a bug however to have a single server that runs (playwright has support for this: https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests but its complicated for us as we have three different apps we would need to route traffic between)

Reply:
Good point. Created this bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2914372.